README.md

```diff
@@ -17,34 +17,34 @@ npm install --save svelte svelte-loader
 Configure inside your `webpack.config.js`:
 
 ```javascript
-  ...
-  resolve: {
-    // see below for an explanation
-    alias: {
-      svelte: path.resolve('node_modules', 'svelte')
-    },
-    extensions: ['.mjs', '.js', '.svelte'],
-    mainFields: ['svelte', 'browser', 'module', 'main']
+...
```

Comment:
I think this was easier to paste into your webpack config when indented since it'd be indented there. I'd probably revert the indentation changes here and below